.github/workflows/test_runner.yml

```diff
@@ -206,3 +218,15 @@ jobs:
     with:
       safe-checkout: ${{ needs.check-tag.outputs.checkout-sha }}
```

Comment:
Missing input?  Or did you skip it because it's not needed?

Reply:
They actually could, maybe that would clean them up a bit.  These staleness tests are kindof a nightmare piece of tech debt I have no idea how to clean up :(. But on presubmit they run the same `blaze query` check N times, where N is the number of release branches.  They only checkout release branches and run tests on them during continuous runs